docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -0,0 +1,170 @@
+
+All you need to do after [recording test runs](/guides/cloud/getting-started) is log in to your Cypress Cloud account and leverage these new debugging capabilities. No additional software, configuration or changes to the existing test recording process is required. For example, you do not need to edit your [Cypress configuration](/guides/references/configuration) file or pass any flags or new commands when recording via [the command line](/guides/guides/command-line#Commands) in order to enable or use Test Replay.
+
+## Accessing Test Replay
+
+Accessing the replay of your test is as simple as reviewing any other test artifact in Cypress Cloud.
+
+1. Hover on a [test for review](/guides/cloud/debugging/recorded-runs#Tests-for-Review) under the Latest Runs Overview.
+
+   <DocsImage
+     src="/img/guides/cloud/test-replay/replay_row_magnified.jpg"
+     alt="Run overview row"
+   />
+
+2. Or navigate to the [test detail sidebar](/guides/cloud/debugging/recorded-runs#Test-detail-sidebar) header or above each attempt in the errors panel. The header option will replay the first failing test while the option above each attempt will replay the affiliated test.
+
+   <DocsImage
+     src="/img/guides/cloud/test-replay/replay_sidebar_magnified.jpg"
+     alt="Run sidebar"
+     width="60%"
+   />
+
+## Debugging With Test Replay
+
+Both options above display the Test Replay interface, which we'll unpack from top to bottom in more detail below.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/replay-demo.gif"
+  alt="Test Replay Demo"
+/>
+
+### Run header
+
+The left side of the header displays the Cypress Cloud test run ID #, with the Git branch name along with the spec, platform and browser information. Hovering on this exposes additional test meta data.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/test-replay-header.jpg"
+  alt="Test Replay header"
+/>
+
+### Command Log
+
+The left side of the screen displays the [command log](/guides/core-concepts/cypress-app#Command-Log) just like in the Cypress app, where you can step through, inspect and [time travel](/guides/core-concepts/cypress-app#Time-traveling) debug your test.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/command-log.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Developer Tools
+
+Dive into network-level traffic, console events and inspect the application under test just as you can in the browser.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/dev-tools.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Replay Controls
+
+Toggle each test attempt, play back errors at a variety of speeds or scrub through the captured error by dragging the playhead along the timeline.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/scrubber.png"
+  alt="Test Replay scrubber controls"
+/>
+
+## Opt out of Test Replay
```

Comment:
We say `Opt-in` for the video config changes in the `MajorVersionWelcome` component: https://github.com/cypress-io/cypress/pull/27549/files#diff-e1690da5e8832a5783a622b238019fa8a00d03899e74bc558b08557ded5c4593R86. I wonder if we should use the same spelling here?
```suggestion
## Opt-out of Test Replay
```